package.json

```diff
@@ -232,7 +232,106 @@
       "@types/mocha": "^10.0.8",
       "rollup@>=4.0.0 <4.22.4": "^4.22.4",
       "@nestjs/common@>=10.0.0 <11.0.0": "10.4.1",
-      "prosemirror-model": "1.22.3"
+      "prosemirror-model": "1.22.3",
```

Comment:
what kind of vulnerabilities are these? and why so many?

Reply:
This should be the result of pnpm audit fix command, https://github.com/novuhq/novu/security/dependabot